tests/phpunit/tests/admin/includesScreen.php

```diff
@@ -155,11 +155,6 @@ class Tests_Admin_includesScreen extends WP_UnitTestCase {
 		),
 	);
 
-	function tearDown() {
-		unset( $GLOBALS['wp_taxonomies']['old-or-new'] );
```

Comment:
The intention of this code is to unregister the taxonomy [which is registered here](https://github.com/johnbillion/wordpress-develop/blob/429ff147337e13d517a2e45b8b1d1eacf8ed3e69/tests/phpunit/tests/admin/includesScreen.php#L250), it's not specifically about the global. There's no need for this because all taxonomies are unregistered in `reset_taxonomies()` which is called by the base `setUp()` method.

Reply:
See my reply above about why doing this reset in `setUp()` is unstable.